### PR TITLE
Fix #169, slightly changed monitor worker processes spawning behavior

### LINKED ISSUE #169: Sometimes SDTmonitor consumer processes crash
Lately some SDTmonitor consumer processes crash abruptly for causes like MemoryError (when the quicklook tries to process in parallel 7 files of ~400MB each). When one of these process crashes, it's not supposed to be spawned again. This leads to a scenery in which less and less consumer processes try to keep up with the new files written onto the disk, miserably failing. I think a better approach would be to spawn a consumer process every time a new file has to be analyzed, with a maximum number of concurrent processes equal to the one chosen via CLI. In this case, if a process crashes unexpectedly, a new process can always be spawned in order to replace it.

### PR DESCRIPTION
**Pull request by giuseppe-carboni**
Also, added a `--plot` argument to SDTpreprocess, the `--debug` argument has been splitted in `--debug`, which basically acts as a `verbose` argument, forcing the process to print debug-related messages, and `--plot`, which forces the process to plot the processed data onto images.

### CODE REVIEW

File: srttools/monitor.py [hunk 1]
```diff
@@ -384,7 +376,7 @@ def create_dummy_config():
     [local]
     [analysis]
     [debugging]
-    debug_file_format : jpg
+    debug_file_format : png
```
Comment (matteobachetti) [inline srttools/monitor.py]: Why? Doesn't it make the page slower to load?

**Comment by matteobachetti** [comment]
I just saw Issue  #171. I see why you converted, but shouldn't it be a matter of installing `pillow`?

**Comment by giuseppe-carboni** [comment]
I'm sorry for all the modifications made to .travis.yml, but I literally have no way to check if they work locally. The CI build should be able to track multiprocessing coverage if the .travis.yml file is updated correctly.

**Comment by giuseppe-carboni** [comment]
No modification to .travis.yml and coveragerc seems to enable multiprocessing coverage tracking. I'm giving up and using Thread objects instead of Process in tests.

**Pull request merged**